### PR TITLE
Bug/search slow

### LINKED ISSUE #307: Proposals ordering issues
It seems proposals are backwards on mainnet under "Execution successful" and so beyond currently active proposals what is shown is the very first proposals in DXdao.
Also currently Expired in Queue is too far up the list. In networks like rinkeby where it happens a lot this is very annoying since new proposals appear very far down the page. Would be great to treat Expired in queue the same as executed.
スクリーンショット 2021-10-26 21 34 33

### PR DESCRIPTION
**Pull request by hhamud**
* Closes #307

### CODE REVIEW

File: src/hooks/useFilteredProposals.ts [hunk 1]
```diff
@@ -13,7 +13,9 @@ const matchScheme = (proposal: ProposalsExtended, scheme: string) =>
   scheme === 'All Schemes' || !scheme ? proposal : proposal.scheme === scheme;
 
 export const useFilteredProposals = () => {
-  const { proposals, loading } = useProposals();
+  const { proposals, loading } = useFilterCriteria();
+  console.log(proposals, loading);
```
Comment (rossneilson) [inline src/hooks/useFilteredProposals.ts]: console

**Comment by rossneilson** [comment]
Abrakadabra and the proposals are gone

Screenshot 2021-11-22 at 19 49 39

Hehe, seems proposals appear for a brief second then disappear?
Filters and searching makes them appear but they are not there by default.

**Review by luchux — APPROVED** [review]
nice!

**Review by luchux — DISMISSED** [review]
nice!

**Comment by rossneilson** [comment]
I see a strange flash effect as we show the initial order first (oldest I think) then it flashes to the most recent,.
Could we change that, its a little jarring

Screenshot 2021-11-23 at 19 04 41

**Review by rossneilson — APPROVED** [review]

**Pull request merged**